Split/Telemetry/Storage/TelemetryConfig.swift

```diff
@@ -64,6 +68,8 @@ struct TelemetryConfig: Codable {
 
     enum CodingKeys: String, CodingKey {
         case operationMode = "oM"
+        case largeSegmentsEnabled = "lsE"
+        case waitForLargeSegments = "wls"
```

Comment:
At the moment, we are not including this config in the telemetry

Reply:
👍

Reply:
Sorry @javrudsky , `wls` field is Okey